### ISSUE TITLE
Duplicated html IDs are created in the dashboard under certain circumstances

### ISSUE THREAD

**Issue by andrewleith** [cds-snc/notification-planning#1251]
## Describe the bug 

When there are sends over more than one month for the sample template, the `/template-usage` page ends up generating duplicate HTML ids:

```
...
<caption class="heading-medium table-heading visuallyhidden">
   June
 </caption>
...
<tr class="table-row" id="cds04145882-0f21-4d57-940d-69883fc23e77">
    ...
    <a class="file-list-filename" href="/services/5c8a0501-2aa8-433a-ba51-cefb8063ab93/templates/04145882-0f21-4d57-940d-69883fc23e77">SMOKE_TEST_EMAIL_BULK</a>     
   ...
</tr>

...

<h2>May</h2>
...
<tr class="table-row" id="cds04145882-0f21-4d57-940d-69883fc23e77">
   ...
   <a class="file-list-filename" href="/services/5c8a0501-2aa8-433a-ba51-cefb8063ab93/templates/04145882-0f21-4d57-940d-69883fc23e77">SMOKE_TEST_EMAIL_BULK</a>        
   ...   
</tr>
...
```

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
See examples in the documentation

SEV-4 Low

## To Reproduce

1. Send emails with the same template over multiple months (or go see the **Cypress UI Tests** service in staging)
2. Click on 'See all templates used'
3. Verify the source of the `<tr>` element used on the same template over 2 months
4. See that they both have the same id

## Expected behavior
- All HTML ids should be unique

## Impact

Describe overall user/system impact to consider when prioritizing this issue.

If applicable

Impact on Notify users:

Impact on Recipients:

Impact on Notify team:

**Comment by yaelberger-commits** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1251?planning-poker) with Zenhub @andrewleith

**Comment by adriannelee** [comment]
Reviewed, will move to done
